improver_tests/utilities/test_OccurrenceWithinVicinity.py

```diff
@@ -312,30 +243,23 @@ def test_with_multiple_times(self):
         expected = np.array(
             [
                 [
-                    [
-                        [0.0, 0.0, 0.0, 0.0],
-                        [1.0, 1.0, 1.0, 0.0],
-                        [1.0, 1.0, 1.0, 0.0],
-                        [1.0, 1.0, 1.0, 0.0],
-                    ],
-                    [
-                        [0.0, 0.0, 1.0, 1.0],
-                        [0.0, 0.0, 1.0, 1.0],
-                        [0.0, 0.0, 1.0, 1.0],
-                        [0.0, 0.0, 0.0, 0.0],
-                    ],
-                ]
+                    [0.0, 0.0, 0.0, 0.0],
+                    [1.0, 1.0, 1.0, 0.0],
+                    [1.0, 1.0, 1.0, 0.0],
+                    [1.0, 1.0, 1.0, 0.0],
+                ],
+                [
+                    [0.0, 0.0, 1.0, 1.0],
+                    [0.0, 0.0, 1.0, 1.0],
+                    [0.0, 0.0, 1.0, 1.0],
+                    [0.0, 0.0, 0.0, 0.0],
+                ],
             ]
         )
-        data = np.zeros((1, 2, 4, 4))
-        data[0, 0, 2, 1] = 1.0
-        data[0, 1, 1, 3] = 1.0
-        cube = set_up_cube(
-            data,
-            "lwe_precipitation_rate",
-            "m s-1",
-            timesteps=np.array([402192.5, 402195.5]),
-        )
+        cube = self.cube[0]
+        cube = add_coordinate(cube, self.timesteps, "time",)
```

Comment:
And again.
```suggestion
        cube = add_coordinate(
            cube,
            self.timesteps,
            "time",
            coord_units="seconds since 1970-01-01 00:00:00",
            dtype=np.int64,
        )
```